content/22.vaccines.md

```diff
@@ -255,9 +258,81 @@ Some common successful whole virus vaccine include targeting influenza viruses,
+This group of 72 participants was split into 2 blocks for low-dose (3 &mu;g) and high-dose (6 &mu;g).
+Within each block, participants were randomly assigned vaccination with CoronaVac or placebo (aluminum diluent without the virus) at a 2:1 ratio.
+Both the vaccine and placebo were prepared in a Good Manufacturing Practice-accredited facility of Sinovac Life Sciences (Beijing, China).
+
+The phase II trial followed the same organization of participants, this time using 300 enrolled participants in the 14-day and another 300 enrolled in the 28-day group.
+One change of note was that the vaccine was produced using a highly automated bioreactor (ReadyToProcess WAVE 25, GE, Umea, Sweden) to increase vaccine production capacity.
+This change resulted in a higher intact spike protein content.
+The authors of this study were not aware of this antigen-level difference between the vaccine batches for the phase I/II when the ethical approval for the trials occurred.
+
+To assess adverse responses, participants were asked to record any events up to 7 days post-treatment. 
+The reported adverse events were graded according to the China National Medical Products Administration guidelines.
+In the phase I trial, the overall incidence of adverse reactions was 29-38% of patients in the 0 to 14 day group and 13-17% in the 0 to 28 day vaccination group.
+The most common symptom was pain at the injection site, which was reported by 17-21% of patients in the 0 to 14 day cohort and 13% in the 0 to 28 day cohort.
+Most adverse reactions were mild (grade 1) where patients recovered within 48 hours.
+There was a single case of acute hypersensitivity with manifestation of urticaria 48 hours following the first dose of study drug was reported in the 6 &mu;g group
+Most adverse reactions were mild (grade 1) in severity and participants recovered within 48 h. 
+There was a single case, from the 6 &mu;g group, of acute hypersensitivity with manifestation of urticaria 48 h after the first dose.
+Both the 14-day and 28-day cohorts had a strong neutralizing Ab response.
+
+In a study with older patients (older than 60 years), they performed another single-center, randomized, double-blind, placebo-controlled, phase I/II trial [@10/fx8z].
+The phase I trial looked at dose escalation using 3 dosages: 1.5, 3 and 6 &mu;g.
+The mean age of participants was 65.8 years (std = 4.8).
+Of 95 screened participants, 72 were enrolled.
+These 72 participants were split into low (3 &mu;g) and high (6 &mu;g) dose groups.
+Within each groups 24 were treated and 12 were placebo.
+A neutralizing antibody response against live SARS-CoV-2 was detected compared to baseline.
+This response was similar across the two dose concentrations.
+Additionally, they did not observe a difference in response between age groups (60–64 years, 65–69 years, and ≥70 years).
+
+In the phase II the mean age was 66.6 years (std = 4.7).
```

Comment:
```suggestion
In phase II the mean age was 66.6 years (standard deviation = 4.7).
```
I'm not sure if that acronym is defined, I haven't seen it used much in proposed text, to my recollection!